apis/nucleus/src/components/listbox/hooks/__tests__/useExistingModel.test.jsx

```diff
@@ -49,20 +49,28 @@ describe('useExistingModel', () => {
 
   afterEach(() => {
     jest.useRealTimers();
-    jest.restoreAllMocks();
     jest.resetAllMocks();
     renderer.unmount();
   });
 
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
```

Comment:
it feels wrong to have this in afterAll as the setup of mocks are in beforeEach (not beforeAll)